twig/twig/2022-09-28.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Possibility to load a template outside a configured directory when using the filesystem loader
+link:      https://symfony.com/blog/twig-security-release-possibility-to-load-a-template-outside-a-configured-directory-when-using-the-filesystem-loader
+cve:       ~
+branches:
+    1.x:
+        time:     2022-09-28 10:36:08
+        versions: ['>=1.0.0', '<1.44.7']
+    2.x:
+        time:     2022-09-28 10:36:08
+        versions: ['>=2.0.0', '<2.15.13']
```

Comment:
@fabpot are you sure about this version ? The latest one on Packagist is 2.15.3

Reply:
That's a typo indeed. Fixed in a506dc8